src/reducers/application.js

```diff
@@ -79,6 +81,18 @@ export default (state = initialState, action) => {
         roles,
       };
     }
+    case SIMPLE_CONFIG_LOADED: {
```

Comment:
It is a bit weird to use redux/saga for initial loading but then local state for the actual form.